ipam/space/space.go

```diff
@@ -122,8 +122,11 @@ func (s *Space) biggestFreeRange(r address.Range) (biggest address.Range) {
 	biggestSize := address.Count(0)
 	s.walkFree(r, func(chunk address.Range) bool {
 		if size := chunk.Size(); size >= biggestSize {
```

Comment:
The biggest chunk does not guarantee that it contains the biggest CIDR-aligned range.